submissions/oluwatosinOke/phpDatabases/customerList.php

```diff
@@ -0,0 +1,84 @@
+<?php
+$servername = "localhost";
+$username = "root";
+$password = "tocyn";
+$dbname = "my_faveorite_online_store";
+
+try {
+  $conn = new PDO("mysql:host=$servername;dbname=$dbname", $username, $password);
```

Comment:
@Tocyn You can have a seperate file for connection. You won't have to repeat this every time but just import where needed.